src/components/show/Show.js

```diff
@@ -0,0 +1,29 @@
+import './show.scss';
+
+export class Show extends Component {
+  state = {
+    btnText: 'Show',
+    show: ''
+  }
```

Comment:
It is better to have true/false flag in state `state = { isActive: false }` and change it `this.setState({ isActive: !this.state.isActive })`
And the in jsx 
```
<button
  className={this.state.isActive ? 'active' : ''}
  onClick={this.showContent}
>
  {this.state.isActive ? 'Hide' : 'Show'}
</button>
...
```